script/test/fixtures/v3/output-k8s-full-example.json

```diff
@@ -160,7 +160,7 @@
         }
       },
       "spec": {
-        "replicas": 1,
+        "replicas": 6,
```

Comment:
but its failing, that's why, changed. but it's `replicas=6` in docker-compose v3 file ?